tests/scripts/github-action-helper.sh

```diff
@@ -294,13 +305,27 @@ function deploy_cluster() {
     echo "invalid argument: $*" >&2
     exit 1
   fi
+
   # enable monitoring
   yq w -i -d0 cluster-test.yaml spec.monitoring.enabled true
   kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
   kubectl create -f monitoring/rbac.yaml
 
-  # create the cluster resources
   kubectl create -f cluster-test.yaml
+
+  deploy_toolbox
+}
+
+# These resources were extracted from the original deploy_cluster(), which was
+# deploying a smorgasbord of resources used by multiple canary tests.  All
+# canary tests using this function should be refactored to only deploy only the
+# minimal set of resources needed.
+function deploy_cluster_full_of_cruft_please_stop_using_this() {
```

Comment:
I'm not sure we want to stop using this entirely, but renaming it to something that suggests caution is good IMO. I think the below is short and should make reasonable devs consider whether they want to deploy everything and the kitchen sink.

```suggestion
function deploy_all_resources() {
```

Reply:
I believe it is poor practice to pull in unnecessary resources and I suspect that causes the canary tests to use additional wallclock time while waiting CRs sync.

I'm negotiable on the function name. :shrug:

Reply:
I've renamed the func to `deploy_all_resources`.

I have had good luck with hyperbolic func names preventing them from continuing to be cut'n'pasted...